countries-of-the-world/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="../css/styles.css">
+</head>
+<body>
+	<header>
+        <nav>
+            <ul>
+                <li><a href="../">Country Lookup</a></li>
+                <li><a href="../countries-of-the-world/" class="active">Countries of the World</a></li>
```

Comment:
De href had `./countries-of-the-world/index.html` moeten zijn. Nu kom ik in de file explorer zegmaar